CMakeLists.txt

```
@@ -212,10 +212,12 @@ if(SDLSOUND_BUILD_STATIC)
     if(NOT MSVC)
         set_target_properties(SDL2_sound-static PROPERTIES OUTPUT_NAME "SDL2_sound")
     endif()
-    target_include_directories(SDL2_sound-static PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>)
+    target_include_directories(SDL2_sound-static PUBLIC 
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
     if(TARGET SDL2::SDL2)
-        target_include_directories(SDL2_sound-static
-            PUBLIC $<TARGET_PROPERTY:SDL2::SDL2,INTERFACE_INCLUDE_DIRECTORIES>)
+        get_target_property(_sdl2_include_dir SDL2::SDL2 INTERFACE_INCLUDE_DIRECTORIES)
+        target_include_directories(SDL2_sound-static PUBLIC "${_sdl2_include_dir}")
```

We want to keep these paths hidden from the installed cmake config files.
```suggestion
        get_target_property(_sdl2_include_dir SDL2::SDL2 INTERFACE_INCLUDE_DIRECTORIES)
        target_include_directories(SDL2_sound-static PUBLIC $<BUILD_INTERFACE:${_sdl2_include_dir}>)
```

---

Done with 83df0f87e9052f32b30c32dba7128a2384296c5c